app/services/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -4,6 +4,7 @@ module Scope
       module Macros
         module Helpers
           include Foreman::Renderer::Errors
+          include ProvisioningTemplatesHelper
```

Comment:
Almost the same question as before. Moreover, we don't include view helpers in macros.

Reply:
May I ask how view helpers are defined? Or better to say, how can I determine if a module / class is a view helper?

Reply:
> how can I determine if a module / class is a view helper

They land in app/helpers and required/included automatically to be used within erb views.

Reply:
Thanks a lot!